compiler/optimizer/VPHandlers.cpp

```diff
@@ -447,6 +446,30 @@ static bool tryFoldCompileTimeLoad(
                }
             return didSomething;
             }
+         else if (node->getSymbolReference() == vp->comp()->getSymRefTab()->findVftSymbolRef()
+                  && node->getFirstChild() == curNode
+                  && constraint->isNonNullObject()
+                  && constraint->getClassType()
+                  && constraint->getClassType()->asFixedClass()
+                  && constraint->getClass())
+            {
+            TR_OpaqueClassBlock *clazz = constraint->getClass();
+#ifdef J9_PROJECT_SPECIFIC
+            TR_J9VMBase *fej9 = static_cast<TR_J9VMBase *>(vp->comp()->fe());
```

Comment:
This looks correct, but I think within `#ifdef J9_PROJECT_SPECIFIC`, it should be possible to call `vp->comp()->fej9()`

Reply:
Changed in https://github.com/eclipse/omr/compare/8066b3dfe38dd091f069482ff4d5daf4cefce3f8..d3b362f26028466e8b392f5c7eaa372926a960c8

Reply:
Sorry, I only mentioned it because it would let you remove the cast

Reply:
I should have removed the cast as well when changed the query to get frontend. Thanks for pointing it out. Changed in https://github.com/eclipse/omr/compare/d3b362f26028466e8b392f5c7eaa372926a960c8..1818cb10df73ff42e54e21dbc0a2246e6e504ae2